internal/app/routers/router-connector.go

```diff
@@ -41,6 +41,7 @@ func ConnectRoutersToHandlers(router *gin.Engine, h dependencies.Handlers) {
 	api.GET("/courses/", h.CourseHandler.GetAllCourses)
 	api.PUT("/courses/", h.AuthHandler.ProtectedEndpoint(), h.CourseHandler.UpdateCourse)
 	api.DELETE("/courses/:id", h.AuthHandler.ProtectedEndpoint(), h.CourseHandler.DeleteCourse)
+	api.POST("/courses/rating", h.CourseHandler.CreateRatingCourse)
```

Comment:
We should add protect middleware , and it gets the user ID from its token. Client does not sends id of user. Id should come from token.You can obtain it like : "userID:=c.MustGet("user_id")" in your next middleware.See in educenter handler .